draft-ietf-rats-corim.md

```diff
@@ -1100,24 +1100,85 @@ The `uint` and `text` types MUST NOT be interpreted in a global scope.
 
 #### Reference Values Triple {#sec-comid-triple-refval}
 
-A Reference Values triple relates reference measurements to a Target
-Environment. For Reference Value Claims, the subject identifies a Target
-Environment, the object contains measurements, and the predicate asserts that
-these are the expected (i.e., reference) measurements for the Target
-Environment.
+The Reference Values Triple has the following structure:
 
 ~~~ cddl
 {::include cddl/reference-triple-record.cddl}
 ~~~
 
+The `reference-triple-record` has the following parameters:
+
+* `ref-env`: Search criteria that target Evidence environments in the ACS.
+* `ref-claims`: Search criteria that target Evidence Claims in the ACS.
+
+To process `reference-triple-record` both the `ref-env` and `ref-claims` are compared with evidence entries from the ACS.
+If these conditions are met, the matching ACS entry is copied into a new ACS entry that is added to the ACS, but with the Reference Value Provider's authority.
+
 #### Endorsed Values Triple {#sec-comid-triple-endval}
 
-An Endorsed Values triple declares additional measurements to add to the ACS.
+The Endorsed Values Triple has the following structure:
 
 ~~~ cddl
 {::include cddl/endorsed-triple-record.cddl}
 ~~~
 
+The `endorsed-triple-record` has the following parameters:
+
+* `condition`: Search criteria applied to the ACS that determine if the `endorsement` applies.
```

Comment:
This section should say:
--> What triple is intended to convey

Versus, 

What CoRIM spec is guiding users to use a possible internal representation in terms of ACS.

This document should be very much applicable and usable to some Supply Chain Consumers, who want to use the Triple based on their semantics but do NOT use Verifier logic which uses ACS but have another means to process the Triple and Implement a Verifier.

I am little concerned that we have moved ACS and internal representation far too much into the base triple description.

Reply:
Is it more clear to say: "Search criteria that targets Attester state."

Reply:
I think we should not shy away from the mechanics here, i.e., linking triples and ACS, because it is necessary for the reader to understand the connection between the two. However, it makes sense to transition from abstract to practical in a smoother way.

Reply:
@thomas-fossati : I tend to dis-agree: I see the clear distinction as below:

Triple should describe what it is meant for : For Ref Triple, used to convey Ref values
for the Env in subject with Reference Measurements/claims in the object. These measurements are used by Verifier to match against the same measurements as received from Attester in Evidence. One possible way of converting these measurements internally and matching logics are described in section : XYZ:

_In another section (XYZ outside this section)_ 
How a potential one use of the triple by converting into a representation best use for matching is where ACS comes into picture!

Reply:
> Is it more clear to say: "Search criteria that targets Attester state."

Yes, at high level, it is fine to say something about matching with Attester Evidence claims